source/core/coverage/ut_coverage.pkb

```diff
@@ -275,23 +165,22 @@ create or replace package body ut_coverage is
     type t_source_lines is table of binary_integer;
     l_source_lines     t_source_lines;
     line_no            binary_integer;
-    l_schema_names     ut_varchar2_list := coalesce(g_schema_names,ut_varchar2_list(sys_context('USERENV','CURRENT_SCHEMA')));
+    l_schema_names     ut_varchar2_list;
     l_query            varchar2(32767);
   begin
+    l_schema_names := coalesce(a_coverage_options.schema_names, ut_varchar2_list(sys_context('USERENV','CURRENT_SCHEMA')));
 
     if not ut_coverage_helper.is_develop_mode() then
       --skip all the utplsql framework objects and all the unit test packages that could potentially be reported by coverage.
-      l_skipped_objects := ut_utils.get_utplsql_objects_list()
-                           multiset union all g_unit_test_packages
-                           multiset union all coalesce(g_exclude_list, ut_object_names());
+      l_skipped_objects := ut_utils.get_utplsql_objects_list() multiset union all coalesce(a_coverage_options.exclude_objects, ut_object_names());
```

Comment:
It was `g_unit_test_packages` here previously, why is it removed from the skipped item list?